public/app/features/backup/components/BackupInventory/BackupInventory.service.ts

```diff
@@ -0,0 +1,54 @@
+import { api } from 'app/percona/shared/helpers/api';
+import { Backup, BackupResponse, DataModel, Status } from './BackupInventory.types';
+
+const BASE_URL = '/v1/management/backup/Artifacts';
+
+export const BackupInventoryService = {
+  async list(): Promise<Backup[]> {
+    return api
+      .post<BackupResponse, any>(`${BASE_URL}/List`, {})
+      .then(
```

Comment:
You used `async` here and used `.then`, why not `await`?

Reply:
thanks 👍